Dockerfile

```diff
@@ -1,10 +1,58 @@
-FROM python:3.10-slim
+FROM debian:10.13-slim as base
 
-RUN useradd -ms /bin/bash user
+WORKDIR /app/src
+
+COPY .nvmrc .
+COPY .babelrc .
+COPY package.json .
+COPY package-lock.json .
+COPY tsconfig.json .
+
+COPY .eslintrc.json .
+COPY .eslintignore .
+
+COPY jest.config.js .
+COPY jest.setup.js .
+
+COPY webpack.config.js .
+
+# https://gist.github.com/remarkablemark/aacf14c29b3f01d6900d13137b21db3a
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y curl && \
+    rm -rf /var/lib/apt/lists/*
+RUN mkdir -p /usr/local/nvm
+ENV NVM_DIR /usr/local/nvm
+ENV NODE_VERSION 16.18.1
+RUN curl --silent -o- https://raw.githubusercontent.com/creationix/nvm/v0.35.1/install.sh | bash
+RUN source $NVM_DIR/nvm.sh \
+    && nvm install $NODE_VERSION \
```

Comment:
why pass env variable instead of using `.nvmrc`?

Reply:
Evaluating `NODE_VERSION` variable moved to `docker build` command. No other option to update `NODE_PATH` and `PATH` dynamically.